app/views/layouts/campaign.html.erb

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Table</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <div class="bg-gray-200">
```

Comment:
Thoughts on extracting the header into a partial shared between the two layouts?

Reply:
Extracted in 5456d350a4dc3cee249a491b6a0ded5f7d330d1d